docs/source/packaging/rpm.rst

```diff
@@ -0,0 +1,63 @@
+==========================
+MOTECH packaging using DEB
+==========================
+
+This is a description of building process for RPM packages on CentOS 7.0
+
+Requirements
+============
+
+* Apache Tomcat 7.0 or greater, can be installed in terminal with "yum" command
+
+.. code-block:: bash
+
+    yum install tomcat.(architecture)
+
+* Java 1.8 or greater, can be installed in terminal with "yum" command
+
+.. code-block:: bash
+
+    yum install java-1.8.0-openjdk-src.(architecture)
+
+* Apache Maven 3 (needed only for building package)
+
+Building process
+================
+
+1) move to motech project directory
+2) change directory to ./packaging/rpm
+3) run maven command with "RMP" profile
+
+.. code-block:: bash
+
+    mvn clean install -PRPM
+
+4) in the folder "./target/rpm/motech-base/RPMS/noarch/" should appear file
+
+- "motech-base_(release number)-SNAPSHOT(some number).noarch.rpm"
```

Comment:
I believe we also a have 'motech' package here as well. The trick is that to build you have to build the entire project with the RPM profile. It will be accessible in the target folder on the parent project itself.